install/firmware.csv

```diff
@@ -273,6 +273,7 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,5921R40176,5.42,2019-11-05T01:07:38Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5921R40183,5.42,2019-09-19T01:15:32Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5921R40189,5.42,2019-10-29T22:31:00Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5921R40190,5.42,2019-10-15T00:08:16Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE ALLOY,5940R40053,5.42,2019-12-03T21:20:38Z,9999-01-01T00:00:00Z,
```

Comment:
This row should be earlier in the file - firmwares.csv is sorted by make > model > s/n